ci-node-modules.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+rmNodeModules()
+{
+	if [ -d "node_modules" ]; then
+		echo "Deleting node_modules"
+		rm -r node_modules
+	else
+		echo "No node_modules folder to delete"
+	fi
+}
+
+if [ -f "yarn.lock" ]; then
+	echo "yarn.lock found."
+	rmNodeModules
+	echo "Running yarn install ..."
+	yarn install
```

Comment:
probably it is better to pass the `--frozen-lockfile` option, so we are sure the lockfile is not modified. That can happen, for example, if `yarn` versions differ. Ref: https://yarnpkg.com/lang/en/docs/cli/install/